### ISSUE TITLE
D1.1 New export wizard

### ISSUE THREAD

**Issue by IanMayo** [debrief/debrief#2933]
As an **analyst** I need to **provide missing metadata** in order to **populate data fields with values not currently present in Debrief**.

We're going to be using a Wizard to do this.  This task covers the generation of that wizard.

Extra detail on the issue is present in the #2912 parent epic.

**Update 4/7/18** - we have some outstanding issues on this, including a couple of missing fields.

**Comment by theanuradha** [comment]
@IanMayo https://github.com/debrief/debrief/pull/2941

**Comment by IanMayo** [comment]
Hi there @theanuradha - I demo'd the new feature to the users this morning.  I got the following feedback:
1). They would like free-text boxes for "Semi-Major Axis" and "Semi-Minor Axis" (_I was mistaken about hard-coding them_)
2) The Classification should allow editing, as well as the drop-down values 
3) The Flag field should allow free editing, in addition to the drop-down vlaues
4) The Type box will be free-text, without any checks.

Note: I've tried to reflect the above in the wizard wireframes:
https://docs.google.com/presentation/d/1jNJ8TqeodjLW7oWzj3JLuqCrfwvT0zOYU_AAWSJshLE/edit#slide=id.g3c02f236c2_0_0
